### PR TITLE
Update Sean's PPP

### PR DESCRIPTION
**Pull request by ExtraShotLatte**
Update Sean's Latest PPP

### CODE REVIEW

File: docs/team/extrashotlatte.md [hunk 1]
```diff
@@ -41,13 +41,14 @@ Given below are my contributions to the project.
...
       * Restructured the presentation of the user guide for clarity
-      (Pull Request [#127](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/127)).
+      (Pull Request [#127](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/127), [#244](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/244)).
       * Added logo to the user guide (Pull Request [#127](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/127)).
       * Added section on how to use the user guide
       (Pull Request [#127](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/127)).
       * Enhanced first-time-user story (Pull Request [#142](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/142)).
       * Added additional command guides and examples (Pull Request [#151](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/151)).
       * Sorted UG and added callouts (Pull Request [#165](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/165), [#177](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/177), [#242](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/242)).
+      * Improved overall friendliness of User Guide (Pull Request [#245](https://github.com/AY2223S2-CS2103T-T11-3/tp/pull/254)).
```
Comment (bryansendeavour) [inline docs/team/extrashotlatte.md:51]: Wanna check is its 245 or 254 in the link?

**Review by bryansendeavour — CHANGES_REQUESTED** [review]
LGTM, just a typo found :)

File: docs/team/extrashotlatte.md (hunk 1 above)
Comment (ExtraShotLatte) [reply docs/team/extrashotlatte.md:51]: It's 245! Thank you for noticing!

**Review by bryansendeavour — APPROVED** [review]
Amazing!

**Pull request merged**